src/components/forms/SelectField/SelectField.js

```diff
@@ -68,6 +85,14 @@ const SelectField = ({
           />
         )}
         {...props}
+        onChange={newValue => {
```

Comment:
Isn't `newValue` and event?

Reply:
it's not because of the API of react-select. it doesn't dispatch any events, but instead handles its internal state that you can tap into via certain callbacks.

![image](https://github.com/open-formulieren/open-forms-sdk/assets/5518550/403c56ce-6016-442d-afa6-46e9347ee19f)